src/scroll-behavior.js

```diff
@@ -0,0 +1,61 @@
+import { wrap } from 'lodash';
+import ScrollBehavior from 'scroll-behavior';
+import Router from 'next/router';
+import Link from 'next/link';
+import setupHistory from './history';
+import StateStorage from './StateStorage';
+
+/* istanbul ignore next*/
+if (typeof window !== 'undefined') {
+    setupHistory();
+}
+
+const stateStorage = new StateStorage();
+const scrollBehaviorSymbol = Symbol('scrollBehavior');
+
+const getScrollBehavior = (options = { disableNextLinkScroll: true }) => {
```

Comment:
```suggestion
const getScrollBehavior = (options) => {
    options = {
        disableNextLinkScroll: true,
        ...options,
    };
```

The reason to do this is that normally `options` means that I can pass an objects with just a **few** keys that I want to change. By using `options = { disableNextLinkScroll: true }`, I would have to pass the full options object, even if I just wanted to change one.

I know that we just have one option and it's not necessary, but it's a good practice and it's ready in case we need further options.

Reply:
agreed ✅